lib/Service/DisplayNotifyService.php

```diff
@@ -219,13 +219,15 @@ public function notifyByDisplayGroupId($displayGroupId)
     }
 
     /** @inheritdoc */
-    public function notifyByCampaignId($campaignId)
+    public function notifyByCampaignId($campaignIds)
```

Comment:
Would it be easier to make it so that we can pass an int or an array into this method?